src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -92,6 +92,13 @@ public function load(array $configs, ContainerBuilder $container)
             $dir = dirname($reflection->getFileName()).'/Resources/views';
             if (is_dir($dir)) {
                 $this->addTwigPath($twigFilesystemLoaderDefinition, $dir, $bundle);
+
+                $classMock = new $class();
+                $parent = $classMock->getParent();
+
+                if ($parent) {
```

Comment:
That makes sense ;-) but do you agree on this approach? Im not sure how much we can rely on the default ordering of `kernel.bundles`, i guess we should correctly order them here for now.